src/ContainerDetails.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import cockpit from 'cockpit';
+
+const _ = cockpit.gettext;
+class ContainerDetails extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+
+    render() {
+        let container = this.props.container;
+        return (
+            <div className='listing-ct-body'>
+                <dl>
+                    <dt>{_("Id")}      </dt> <dd>{ container.ID }</dd>
```

Comment:
Please avoid the extra whitespace in the element. Perhaps they are not rendered, but it might still cause some layout issues. Also please be consistent with putting the `<dd>`s into a new line.

"Id" should be spelled "ID", as it's an acronym.